packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1054,6 +1062,18 @@ export class MergeTree {
 			const firstMoveSeqIdx = this.moveSeqs.findIndex((seq) => seq >= minSeq);
 			this.moveSeqs = firstMoveSeqIdx === -1 ? [] : this.moveSeqs.slice(firstMoveSeqIdx);
 			if (MergeTree.options.zamboniSegments) {
+				walkAllChildSegments(this.root, (seg) => {
```

Comment:
this is really costly, and would impact all users of merge tree, not just obliterate.  we need to find a way to accelerate this. we'll likely have to store all the obliterate infos somewhere, so we can quickly look up like we do for moveSeqs and just remove those below the updated minseq